internal/pkg/service/buffer/worker/service/file_import.go

```diff
@@ -81,32 +81,22 @@ func (s *Service) importFiles(ctx context.Context, wg *sync.WaitGroup, d depende
 						return "", err
 					}
 					return "skipped import of the empty file", nil
-				}
+				} else {
```

Comment:
Preco je tu `else` ked v podmienke vyssie je `return`?

Reply:
Pride mi lepsie tu ten `else` nemat.

Priklad ak by tych "specialnych" situacii bolo viac:
```
if {
    // 1
} else {
    if {
         // 2
    } else {
        if {
             // 3
        } else {
             // else
        }
    }
}
```

Vs:
```
if {
// 1
return
}

if {
// 2
return
}

if {
// 3
return
}

// else
```

Reply:
Já jsem si toho `return` poprvé nevšimnul, když jsem to četl a divil jsem se, proč je tam stejný kód dvakrát.

Reply:
To, ze tam bol kod 2x si fixol dobre, to je ina vec.

Reply:
Souvisí to s tím, ta indentace mi přijde přehlednejší, když je to "tail end" té funkce a nejedná se o guard condition. Takhle hned na první pohled z toho poznám, že jsou to dva branche:

```
if condition() {
  // a
  // b
  // c
  return
} else {
  // a
  // c
  // b
  // d
  return
}
```

Taky tě pak compiler donutí mít v obou branchích `return`, ale to je asi hodně minor benefit.

Reply:
Ako ten jeden `else` tam extra nevadi, len som chcel upozornit, ze je lepsie sa vyhnut "vnaraniu", ked sa da.

Reply:
Mne to pride divne, ale pri jednej podmienke je to asi iba vec nazoru. Dik za vysvetlenie.